src/templates/bash-template

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# Copyright 2020 AstroLab Software
+# Author: Yash Datta
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+if [ $# -le 1 ]; then
+  echo "Usage: --config <config-file>"
+  exit 1
+fi
+
+VARS=$@
+
+# Parse config and rules file paths
+while test $# -gt 0
+do
+  case "$1" in
+    --config)
+      shift
+      CONF_FILE_PATH=$1;;
+  esac
+  shift
+done
+
+# Convert file paths to their basenames, since spark just needs base names to refer to these files
+CONF_FILE=$(basename $CONF_FILE_PATH)
+
+VARS=${VARS/$CONF_FILE_PATH/$CONF_FILE}
+
+lib_dir=$(cd "$(dirname ${BASH_SOURCE[0]})"/..; pwd)
```

Comment:
According to the README, one should run `./bin/start.sh --config conf/application.conf`, hence `lib_dir` must be:

```bash
lib_dir=$(cd "$(dirname ${BASH_SOURCE[0]})"/../lib; pwd)
```

(otherwise I have path mismatch).

Reply:
thanks for pointing this out, i forgot to push changes to the start script since it is generated from the bash template